documentation/practices.md

```diff
@@ -52,9 +39,28 @@ If you use another CMS, you must update the configurations to reflect the public
 
 **IBM Watson Content Hub (WCH) Integration**
 
-IBM Watson Content Hub is a cloud-based CMS with an embedded content delivery network that lets marketers update content faster. It also provides services with IBM Watson such as cognitive tagging, to help transform assets into a searchable library of content.
-Catalog images and site content images can be uploaded to WCH using the wchtools command-line utility. Uploaded assets are scanned using Watson's Visual Image recognition service, and tagged based on content to create a searchable library. After images are uploaded, the content delivery url must be provided in the reference storefront's configuration for both the skuImagesUrl and siteImagesUrl with the appropriate sku/file-name placeholders. For example, certain fields in the following examples are populated based on the tenant ID of the WCH account being used, and the directory structure of the assets uploaded.
+IBM Watson Content Hub is a cloud-based CMS with an embedded content delivery network that lets marketers update content faster. It also provides services with IBM Watson, such as cognitive tagging, to help transform assets into a searchable library of content.
+
+Catalog images and site content images can be uploaded to WCH using the wchtools command-line utility. Uploaded assets are scanned using Watson's Visual Image recognition service, and tagged based on content to create a searchable library. After the images are uploaded, the content delivery URL must be provided in the Reference Storefront's configuration for both the `skuImagesUrl` and `siteImagesUrl` with the appropriate sku/file-name placeholders. For example, certain fields in the following examples are populated based on the tenant ID of the WCH account being used, and the directory structure of the assets uploaded.
```

Comment:
Catalog images and site content images can be uploaded to WCH using the wchtools command-line utility.
  - Paasive
     Correct : You can upload catalog images and site content images to WCH using the wchtools command-line utility.
       

Other versions for future reference: 

- With wchtools command-line utility, you can upload catalog images and site content images to WCH (for a conceptual topic that talks about this feature) 

- Use wchtools command-line utility to upload catalog images and site content images to WCH (for a task on how to use this)

Change rest of the section as well.

In the following examples, a few fields are populated based on the tenant ID of the WCH account and the directory structure of the assets: